console/src/config.rs

```diff
@@ -28,7 +29,17 @@ pub struct Config {
 
     #[clap(flatten)]
     pub(crate) view_options: ViewOptions,
+
+    /// How long to continue displaying completed tasks and dropped resources
+    /// after they have been closed. Accepted values are:
+    ///
+    /// - durations in the form of 5days 2min 2s.
+    /// - `none` to disable this functionality
```

Comment:
```suggestion
i wonder if it's worth explicitly saying something like "
```suggestion
    /// - Durations in the form of `5days 2min 2s` and similar. Durations are parsed using
    ///    the [`humantime` crate's `parse_duration` function][1]. See that crate's documentation
    ///    for details.
    /// - `none` to disable this functionality
    /// [1]: https://docs.rs/humantime/2.1.0/humantime/fn.parse_duration.html
```
or, alternatively, we could just coy from their docs:
```suggestion
    /// - Durations, parsed as a combination of time spans (such as `5days 2min 2s`). 
    ///    
    ///   Each time span is an integer number followed by a suffix. Supported suffixes are:
    ///
    ///   * `nsec`, `ns` -- nanoseconds
    ///   * `usec`, `us` -- microseconds
    ///   * `msec`, `ms` -- milliseconds
    ///   * `seconds`, `second`, `sec`, `s`
    ///   * `minutes`, `minute`, `min`, `m`
    ///   * `hours`, `hour`, `hr`, `h`
    ///   * `days`, `day`, `d`
    ///   * `weeks`, `week`, `w`
    ///   * `months`, `month`, `M` -- defined as 30.44 days
    ///   * `years`, `year`, `y` -- defined as 365.25 days
    /// - `none` to disable removing completed task spans
```

(i don't know how `clap`'s derive macro handles markdown links in RustDoc, so if we want to link to their docs, we may be better off just including the actual URL)